// @primer/react-brand/lib/design-tokens/js/tokens/functional/components/testimonial/colors.js
/**
 * Do not edit directly
 * Generated on Wed, 13 Mar 2024 11:00:46 GMT
 */

export const BrandTestimonialQuoteMarkColorDefault = "var(--brand-color-text-default)";
export const BrandTestimonialQuoteMarkColorBlue = "var(--base-color-scale-blue-5)";
export const BrandTestimonialQuoteMarkColorBlue-purpleStart = "var(--base-color-scale-blue-5)";
export const BrandTestimonialQuoteMarkColorBlue-purpleEnd = "var(--base-color-scale-purple-5)";
export const BrandTestimonialQuoteMarkColorCoral = "var(--base-color-scale-coral-5)";
export const BrandTestimonialQuoteMarkColorGreen = "var(--base-color-scale-green-5)";
export const BrandTestimonialQuoteMarkColorGreen-blueStart = "var(--base-color-scale-green-5)";
export const BrandTestimonialQuoteMarkColorGreen-blueEnd = "var(--base-color-scale-blue-5)";
export const BrandTestimonialQuoteMarkColorGray = "var(--base-color-scale-gray-5)";
export const BrandTestimonialQuoteMarkColorIndigo = "var(--base-color-scale-indigo-5)";
export const BrandTestimonialQuoteMarkColorLemon = "var(--base-color-scale-lemon-5)";
export const BrandTestimonialQuoteMarkColorLime = "var(--base-color-scale-lime-5)";
export const BrandTestimonialQuoteMarkColorOrange = "var(--base-color-scale-orange-5)";
export const BrandTestimonialQuoteMarkColorPink = "var(--base-color-scale-pink-5)";
export const BrandTestimonialQuoteMarkColorPink-blueStart = "var(--base-color-scale-pink-5)";
export const BrandTestimonialQuoteMarkColorPink-blueEnd = "var(--base-color-scale-blue-5)";
export const BrandTestimonialQuoteMarkColorPurple = "var(--base-color-scale-purple-5)";
export const BrandTestimonialQuoteMarkColorPurple-redStart = "var(--base-color-scale-purple-5)";
export const BrandTestimonialQuoteMarkColorPurple-redEnd = "var(--base-color-scale-red-5)";
export const BrandTestimonialQuoteMarkColorRed = "var(--base-color-scale-red-5)";
export const BrandTestimonialQuoteMarkColorRed-orangeStart = "var(--base-color-scale-red-5)";
export const BrandTestimonialQuoteMarkColorRed-orangeEnd = "var(--base-color-scale-orange-5)";
export const BrandTestimonialQuoteMarkColorTeal = "var(--base-color-scale-teal-5)";
export const BrandTestimonialQuoteMarkColorYellow = "var(--base-color-scale-yellow-5)";
