// @primer/react-brand/lib/design-tokens/js/tokens/functional/size/size.js
/**
 * Do not edit directly
 * Generated on Wed, 13 Mar 2024 11:00:46 GMT
 */

export const BrandControlMinTargetFine = "1rem";
export const BrandControlMinTargetCoarse = "2.75rem";
export const BrandControlSmallSize = "2rem";
export const BrandControlSmallLineBoxHeight = "0.75rem";
export const BrandControlSmallPaddingBlock = "0.75rem";
export const BrandControlSmallPaddingInlineCondensed = "0.5rem";
export const BrandControlSmallPaddingInlineNormal = "0.75rem";
export const BrandControlSmallPaddingInlineSpacious = "1.5rem";
export const BrandControlSmallGap = "0.5rem";
export const BrandControlMediumSize = "3rem";
export const BrandControlMediumLineBoxHeight = "1rem";
export const BrandControlMediumPaddingBlock = "1rem";
export const BrandControlMediumPaddingInlineCondensed = "0.75rem";
export const BrandControlMediumPaddingInlineNormal = "1rem";
export const BrandControlMediumPaddingInlineSpacious = "1.75rem";
export const BrandControlMediumGap = "0.75rem";
export const BrandControlLargeSize = "3.75rem";
export const BrandControlLargeLineBoxHeight = "1.25rem";
export const BrandControlLargePaddingBlock = "1.25rem";
export const BrandControlLargePaddingInlineCondensed = "1rem";
export const BrandControlLargePaddingInlineNormal = "1.25rem";
export const BrandControlLargePaddingInlineSpacious = "2rem";
export const BrandControlLargeGap = "1rem";
export const BrandControlAnimationDuration = "80ms";
export const BrandControlAnimationEasing = "var(--brand-animation-easing-glide)";
export const BrandBoxSpacingCondensed = "1rem";
export const BrandBoxSpacingNormal = "1.5rem";
export const BrandBoxSpacingSpacious = "3rem";
export const BrandStackPaddingCondensed = "1rem";
export const BrandStackPaddingNormal = "1.5rem";
export const BrandStackPaddingSpacious = "3rem";
export const BrandStackGapCondensed = "1rem";
export const BrandStackGapNormal = "1.5rem";
export const BrandStackGapSpacious = "3rem";
export const BrandControlStackSmallGapCondensed = "0.5rem";
export const BrandControlStackSmallGapSpacious = "1rem";
export const BrandControlStackMediumGapCondensed = "0.5rem";
export const BrandControlStackMediumGapSpacious = "0.75rem";
export const BrandControlStackLargeGapAuto = "0.5rem";
export const BrandControlStackLargeGapCondensed = "0.5rem";
export const BrandControlStackLargeGapSpacious = "0.75rem";
