// @primer/react-brand/lib/design-tokens/js/tokens/functional/components/card/colors.js
/**
 * Do not edit directly
 * Generated on Wed, 13 Mar 2024 11:00:46 GMT
 */

export const BrandCardBackgroundDefault = "var(--brand-color-canvas-default)";
export const BrandCardIconBackgroundDefault = "var(--base-color-scale-gray-0)";
export const BrandCardIconBackgroundBlue = "var(--base-color-scale-blue-0)";
export const BrandCardIconBackgroundCoral = "var(--base-color-scale-coral-0)";
export const BrandCardIconBackgroundGreen = "var(--base-color-scale-green-0)";
export const BrandCardIconBackgroundGray = "var(--base-color-scale-gray-0)";
export const BrandCardIconBackgroundIndigo = "var(--base-color-scale-indigo-0)";
export const BrandCardIconBackgroundLemon = "var(--base-color-scale-lemon-0)";
export const BrandCardIconBackgroundLime = "var(--base-color-scale-lime-0)";
export const BrandCardIconBackgroundOrange = "var(--base-color-scale-orange-0)";
export const BrandCardIconBackgroundPink = "var(--base-color-scale-pink-0)";
export const BrandCardIconBackgroundPurple = "var(--base-color-scale-purple-0)";
export const BrandCardIconBackgroundRed = "var(--base-color-scale-red-0)";
export const BrandCardIconBackgroundTeal = "var(--base-color-scale-teal-0)";
export const BrandCardIconBackgroundYellow = "var(--base-color-scale-yellow-0)";
export const BrandCardIconColorDefault = "var(--brand-color-scale-gray-5)";
export const BrandCardIconColorBlue = "var(--base-color-scale-blue-5)";
export const BrandCardIconColorCoral = "var(--base-color-scale-coral-5)";
export const BrandCardIconColorGreen = "var(--base-color-scale-green-5)";
export const BrandCardIconColorGray = "var(--base-color-scale-gray-6)";
export const BrandCardIconColorIndigo = "var(--base-color-scale-indigo-5)";
export const BrandCardIconColorLemon = "var(--base-color-scale-lemon-5)";
export const BrandCardIconColorLime = "var(--base-color-scale-lime-5)";
export const BrandCardIconColorOrange = "var(--base-color-scale-orange-5)";
export const BrandCardIconColorPink = "var(--base-color-scale-pink-5)";
export const BrandCardIconColorPurple = "var(--base-color-scale-purple-5)";
export const BrandCardIconColorRed = "var(--base-color-scale-red-5)";
export const BrandCardIconColorTeal = "var(--base-color-scale-teal-5)";
export const BrandCardIconColorYellow = "var(--base-color-scale-yellow-5)";
