// @primer/react-brand/lib/design-tokens/js/tokens/functional/components/label/colors.js
/**
 * Do not edit directly
 * Generated on Wed, 13 Mar 2024 11:00:46 GMT
 */

export const BrandLabelColorDefault = "var(--brand-color-text-default)";
export const BrandLabelColorBlue = "var(--base-color-scale-blue-5)";
export const BrandLabelColorBlue-purpleStart = "var(--base-color-scale-blue-5)";
export const BrandLabelColorBlue-purpleEnd = "var(--base-color-scale-purple-5)";
export const BrandLabelColorCoral = "var(--base-color-scale-coral-5)";
export const BrandLabelColorGreen = "var(--base-color-scale-green-5)";
export const BrandLabelColorGreen-blueStart = "var(--base-color-scale-green-5)";
export const BrandLabelColorGreen-blueEnd = "var(--base-color-scale-blue-5)";
export const BrandLabelColorGray = "var(--base-color-scale-gray-5)";
export const BrandLabelColorIndigo = "var(--base-color-scale-indigo-5)";
export const BrandLabelColorLemon = "var(--base-color-scale-lemon-5)";
export const BrandLabelColorLime = "var(--base-color-scale-lime-5)";
export const BrandLabelColorOrange = "var(--base-color-scale-orange-5)";
export const BrandLabelColorPink = "var(--base-color-scale-pink-5)";
export const BrandLabelColorPink-blueStart = "var(--base-color-scale-pink-5)";
export const BrandLabelColorPink-blueEnd = "var(--base-color-scale-blue-5)";
export const BrandLabelColorPurple = "var(--base-color-scale-purple-5)";
export const BrandLabelColorPurple-redStart = "var(--base-color-scale-purple-5)";
export const BrandLabelColorPurple-redEnd = "var(--base-color-scale-red-5)";
export const BrandLabelColorRed = "var(--base-color-scale-red-5)";
export const BrandLabelColorRed-orangeStart = "var(--base-color-scale-red-5)";
export const BrandLabelColorRed-orangeEnd = "var(--base-color-scale-orange-5)";
export const BrandLabelColorTeal = "var(--base-color-scale-teal-5)";
export const BrandLabelColorYellow = "var(--base-color-scale-yellow-5)";
