// @primer/react-brand/lib/design-tokens/js/tokens/functional/typography/typography.js
/**
 * Do not edit directly
 * Generated on Wed, 13 Mar 2024 11:00:46 GMT
 */

export const BrandFontStackSystem = "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandFontStackSansSerif = "'Mona Sans', 'MonaSansFallback',-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandFontStackSansSerifAlt = "'Hubot Sans',-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandFontStackMonospace = "ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace";
export const BrandBodyFontFamily = "'Mona Sans', 'MonaSansFallback',-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandHeadingFontFamily = "'Mona Sans', 'MonaSansFallback',-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandHeadingWeight400 = "var(--base-text-weight-semibold)";
export const BrandHeadingWeight500 = "var(--base-text-weight-semibold)";
export const BrandHeadingWeight600 = "var(--base-text-weight-semibold)";
export const BrandHeadingWeight700 = "var(--base-text-weight-semibold)";
export const BrandHeadingLetterSpacing400 = "-0.01em";
export const BrandHeadingLetterSpacing500 = "-0.01em";
export const BrandHeadingLetterSpacing600 = "-0.02em";
export const BrandHeadingLetterSpacing700 = "-0.02em";
export const BrandTextWeight100 = "var(--base-text-weight-normal)";
export const BrandTextWeight200 = "var(--base-text-weight-normal)";
export const BrandTextWeight300 = "var(--base-text-weight-normal)";
export const BrandTextWeight400 = "var(--base-text-weight-semibold)";
export const BrandTextWeight500 = "var(--base-text-weight-semibold)";
export const BrandTextWeight600 = "var(--base-text-weight-semibold)";
export const BrandTextWeight700 = "var(--base-text-weight-semibold)";
export const BrandTextWeight800 = "var(--base-text-weight-semibold)";
export const BrandTextWeight900 = "var(--base-text-weight-semibold)";
export const BrandTextWeight1000 = "var(--base-text-weight-bold)";
export const BrandTextSize100 = "0.875rem";
export const BrandTextSize200 = "1rem";
export const BrandTextSize300 = "1.125rem";
export const BrandTextSize400 = "1.25rem";
export const BrandTextSize500 = "1.375rem";
export const BrandTextSize600 = "1.5rem";
export const BrandTextSize700 = "1.75rem";
export const BrandTextSize800 = "2rem";
export const BrandTextSize900 = "2.25rem";
export const BrandTextSize1000 = "2.5rem";
export const BrandTextLineHeight100 = "1.5";
export const BrandTextLineHeight200 = "1.5";
export const BrandTextLineHeight300 = "1.5";
export const BrandTextLineHeight400 = "1.3";
export const BrandTextLineHeight500 = "1.3";
export const BrandTextLineHeight600 = "1.2";
export const BrandTextLineHeight700 = "1.2";
export const BrandTextLineHeight800 = "1.2";
export const BrandTextLineHeight900 = "1.1";
export const BrandTextLineHeight1000 = "1.1";
export const BrandTextLetterSpacing400 = "-0.01em";
export const BrandTextLetterSpacing500 = "-0.01em";
export const BrandTextLetterSpacing600 = "-0.02em";
export const BrandTextLetterSpacing700 = "-0.02em";
export const BrandTextLetterSpacing800 = "-0.02em";
export const BrandTextLetterSpacing900 = "-0.02em";
export const BrandTextLetterSpacing1000 = "-0.03em";
export const BrandTextStyleItalic1 = "'ital' 1";
export const BrandTextStyleItalic2 = "'ital' 2";
export const BrandTextStyleItalic3 = "'ital' 3";
export const BrandTextStyleItalic4 = "'ital' 4";
export const BrandTextStyleItalic5 = "'ital' 5";
export const BrandTextStyleItalic6 = "'ital' 6";
export const BrandTextStyleItalic7 = "'ital' 7";
export const BrandTextStyleItalic8 = "'ital' 8";
export const BrandTextStyleItalic9 = "'ital' 9";
export const BrandTextStyleItalic10 = "'ital' 10";
