// @primer/react-brand/lib/design-tokens/js/tokens/functional/typography/typography-responsive.js
/**
 * Do not edit directly
 * Generated on Wed, 13 Mar 2024 11:00:46 GMT
 */

export const BrandBodyFontFamilyAlt = "'Hubot Sans',-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandHeadingFontFamilyAlt = "'Hubot Sans',-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const BrandHeadingWeight800 = "var(--base-text-weight-semibold)";
export const BrandHeadingWeight900 = "var(--base-text-weight-semibold)";
export const BrandHeadingWeight1000 = "var(--base-text-weight-bold)";
export const BrandHeadingLetterSpacing800 = "-0.02em";
export const BrandHeadingLetterSpacing900 = "-0.02em";
export const BrandHeadingLetterSpacing1000 = "-0.03em";
export const BrandHeadingLineHeight400 = "1.3";
export const BrandHeadingLineHeight500 = "1.3";
export const BrandHeadingLineHeight600 = "1.3";
export const BrandHeadingLineHeight700 = "1.2";
export const BrandHeadingLineHeight800 = "1.2";
export const BrandHeadingLineHeight900 = "1.1";
export const BrandHeadingLineHeight1000 = "1.1";
export const BrandTextSubheadSizeMedium = "1rem";
export const BrandTextSubheadSizeLarge = "1.125rem";
export const BrandTextSubheadWeightMedium = "var(--base-text-weight-semibold)";
export const BrandTextSubheadWeightLarge = "var(--base-text-weight-semibold)";
export const BrandTextSubheadLineHeightMedium = "1.3";
export const BrandTextSubheadLineHeightLarge = "1.3";
export const BrandTextLetterSpacing100 = "0.02em";
export const BrandTextLetterSpacing200 = "0em";
export const BrandTextLetterSpacing300 = "0em";
